packages/notification-center/src/store/notifications-provider.context.tsx

```diff
@@ -29,10 +30,10 @@ export function NotificationsProvider({ children }: { children: React.ReactNode
 
     if (isRefetch) {
       notifications[storeId] = newNotifications;
-      setNotifications(notifications);
+      setNotifications(Object.assign({}, notifications));
```

Comment:
I did here the assign in order to force re-render on the set notifications, which were missing.

Reply:
and `{...notifications}` do not work?

Reply:
Mostly a preference on my side to use spread in this case 😄

Reply:
@davidsoderberg I am not sure I tried it, does it have other benefits besides the visual one?

Reply:
Nothing that we really need to care about here so it is pure visual :)

Reply:
Well anyway it won't work because it is a Map object 😅